src/App/Fossa/ManualDeps.hs

```diff
@@ -147,8 +164,27 @@ data CustomDependency = CustomDependency
   { customName :: Text
   , customVersion :: Text
   , customLicense :: Text
-  , customDescription :: Maybe Text
-  , customUrl :: Maybe Text
+  , customMetadata :: Maybe CustomDependencyMetadata
+  }
+  deriving (Eq, Ord, Show)
+
+data CustomDependencyMetadata = CustomDependencyMetadata
```

Comment:
[nit]: Should we combine the types of metadata here?  Are `CustomDependencyMetadata` and `RemoteDependencyMetadata` going to diverge?

Reply:
Also, should `VendoredDependency` also support description and homepage?

Reply:
It looks like `VendoredDependency`'s are just converted into locators. I think supporting metadata for them would involve a larger scale refactor so I'm gonna leave that alone for now